lang/julia/Portfile

```diff
@@ -8,7 +8,7 @@ compilers.choose    fc f77 f90
 compilers.setup     require_fortran -g95
 
 github.setup        JuliaLang julia 1.2.0 v
-revision            0
+revision            1
```

Comment:
Revision bump is not required, as this does not change the installed files.

Reply:
Done.